tools/migration_generator/migration_generator.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"text/template"
+	"time"
+
+	"github.com/iancoleman/strcase"
+)
+
+func main() {
+	var migrationName string
+	flag.StringVar(&migrationName, "name", "", "The migration name to use")
+	flag.Parse()
+
+	if migrationName == "" {
+		println("A migration name must be specified")
+		flag.PrintDefaults()
+	}
+
+	now := time.Now().UTC()
+	id := now.Format("20060102150405")
+	funcName := strcase.ToLowerCamel(migrationName)
+	funcNameSnake := strcase.ToSnake(migrationName)
+
+	filePath := filepath.Join("internal/migrations", fmt.Sprintf("%s_%s.go", id, funcNameSnake))
+	testFilePath := filepath.Join("internal/migrations", fmt.Sprintf("%s_%s_test.go", id, funcNameSnake))
+
+	data := struct {
+		ID            string
+		FuncName      string
+		FuncNameSnake string
+	}{
+		ID:            id,
+		FuncName:      funcName,
+		FuncNameSnake: funcNameSnake,
+	}
+
+	file, err := os.Create(filePath)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Failed to create migration file %s: %s", filePath, err)
+		os.Exit(1)
+	}
+
+	testFile, err := os.Create(testFilePath)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Failed to create migration test file %s: %s", testFilePath, err)
+		os.Exit(1)
+	}
+
+	err = template.Must(template.New("migration").Parse(migrationTemplate)).Execute(file, data)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Failed to write migration file %s: %s", filePath, err)
```

Comment:
fmt.Fprintf(os.Stderr, "Failed to write migration file %s: %s\n", filePath, err)